src/table/io/FitsSchemaInputMapper.cc

```diff
@@ -136,13 +136,10 @@ FitsSchemaInputMapper::FitsSchemaInputMapper(daf::base::PropertyList &metadata,
         // Read slots saved using an old mechanism in as aliases, since the new slot mechanism delegates
         // slot definition to the AliasMap.
         static std::array<std::pair<std::string, std::string>, 7> oldSlotKeys = {
-                std::make_pair("PSF_FLUX", "slot_PsfFlux"),
-                std::make_pair("AP_FLUX", "slot_ApFlux"),
-                std::make_pair("INST_FLUX", "slot_InstFlux"),
-                std::make_pair("MODEL_FLUX", "slot_ModelFlux"),
-                std::make_pair("CALIB_FLUX", "slot_CalibFlux"),
-                std::make_pair("CENTROID", "slot_Centroid"),
-                std::make_pair("SHAPE", "slot_Shape")};
+                {std::make_pair("PSF_FLUX", "slot_PsfFlux"), std::make_pair("AP_FLUX", "slot_ApFlux"),
+                 std::make_pair("INST_FLUX", "slot_InstFlux"), std::make_pair("MODEL_FLUX", "slot_ModelFlux"),
+                 std::make_pair("CALIB_FLUX", "slot_CalibFlux"), std::make_pair("CENTROID", "slot_Centroid"),
+                 std::make_pair("SHAPE", "slot_Shape")}};
```

Comment:
My format file looks the same as yours, except with a `---` as the first line, and `...` as the last line (copied directly from the [dev-guide](https://developer.lsst.io/tools/clang_format.html)).

I just re-ran clang-format outside of sublimeText, and it didn't change the file as checked-in. My version is also installed via homebrew.